src/Fixer/Whitespace/StatementIndentationFixer.php

```diff
@@ -174,13 +185,15 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
 
         /**
          * @var list<array{
-         *     type: 'block'|'block_signature'|'statement',
+         *     type: 'block'|'block_signature'|'statement'|'array'|'array_access',
          *     skip: bool,
          *     end_index: int,
          *     end_index_inclusive: bool,
          *     initial_indent: string,
+         *     base_indent: string,
```

Comment:
What's the difference between `initial_indent` and `base_indent` (conceptually)? I believe that in more than half of the places these have the same value, only a few later they differ. Since the name is quite similar, I would like to understand it better.

Reply:
If I remember correctly, `initial_indent` is the indent of the first line of a scope before it gets fixed, and `base_indent` is the minimum indent that scope requires for its content.